src/plugins/unified_histogram/public/chart/get_lens_attributes.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { DataView, DataViewField } from '@kbn/data-views-plugin/public';
+import type { AggregateQuery, Filter, Query } from '@kbn/es-query';
+import { i18n } from '@kbn/i18n';
+import type {
+  CountIndexPatternColumn,
+  DateHistogramIndexPatternColumn,
+  GenericIndexPatternColumn,
+  TermsIndexPatternColumn,
+  TypedLensByValueInput,
+} from '@kbn/lens-plugin/public';
+import { fieldSupportsBreakdown } from './field_supports_breakdown';
+
+export const getLensAttributes = ({
+  filters,
+  query,
+  dataView,
+  timeInterval,
+  breakdownField,
+}: {
+  filters: Filter[];
+  query: Query | AggregateQuery;
+  dataView: DataView;
+  timeInterval: string | undefined;
+  breakdownField: DataViewField | undefined;
+}) => {
+  const showBreakdown = breakdownField && fieldSupportsBreakdown(breakdownField);
+
+  let columnOrder = ['date_column', 'count_column'];
+
+  if (showBreakdown) {
+    columnOrder = ['breakdown_column', ...columnOrder];
+  }
+
+  let columns: Record<string, GenericIndexPatternColumn> = {
+    date_column: {
+      dataType: 'date',
+      isBucketed: true,
+      label: dataView.timeFieldName ?? '',
+      operationType: 'date_histogram',
+      scale: 'interval',
+      sourceField: dataView.timeFieldName,
+      params: {
+        interval: timeInterval ?? 'auto',
+      },
+    } as DateHistogramIndexPatternColumn,
+    count_column: {
+      dataType: 'number',
+      isBucketed: false,
+      label: i18n.translate('unifiedHistogram.countColumnLabel', {
+        defaultMessage: 'Count of records',
+      }),
+      operationType: 'count',
+      scale: 'ratio',
+      sourceField: '___records___',
+      params: {
+        format: {
+          id: 'number',
+          params: {
+            decimals: 0,
+          },
+        },
+      },
+    } as CountIndexPatternColumn,
+  };
+
+  if (showBreakdown) {
+    columns = {
+      ...columns,
+      breakdown_column: {
+        dataType: 'string',
+        isBucketed: true,
+        label: i18n.translate('unifiedHistogram.breakdownColumnLabel', {
+          defaultMessage: 'Top 3 values of {fieldName}',
+          values: { fieldName: breakdownField?.displayName },
+        }),
+        operationType: 'terms',
+        scale: 'ordinal',
+        sourceField: breakdownField.name,
+        params: {
+          size: 3,
+          orderBy: {
+            type: 'column',
+            columnId: 'count_column',
+          },
+          orderDirection: 'desc',
+          otherBucket: false,
```

Comment:
@davismcphee @flash1293  here's where the `otherBucket` needs to be enabled, right (tried locally, but didn't work, guess it's more work then that)

@flash1293 I'm interested what the performance impact would be ?

Reply:
The impact is another request after the first one completes, adding an "Other" series to the histogram

Reply:
Ok I seen, I think it would be more accurate this way

Reply:
I originally did include the 'other' bucket and can add it back in, but I removed it since none of the mockups we were working with included it: https://www.figma.com/file/DYnliF6FgvQTxUTwctMo1B/Enhanced-histogram. I wonder if this was just an oversight or intentional? I think it's worth discussing with the team to decide.

Reply:
@flash1293 we decided to add the other bucket, it's already implemented